### DISCUSSION TITLE
Office_S_Tips/system/2019/09/20/drobne_podpowiedzi_2

### DISCUSSION THREAD

**Discussion by giscus** [andrzejQ/Office_S_Tips#21] [Komentarze]
# Office_S_Tips/system/2019/09/20/drobne_podpowiedzi_2

Tekst z komputera przez QR-kod do tel. kom. * Po pierwszej instalacji MS Office * Hurtowe zmniejszanie dużych obrazów *.jpg * Problem ogromnego rozmiaru plików RTF i DOC

https://andrzejq.github.io/Office_S_Tips/system/2019/09/20/drobne_podpowiedzi_2.html

**Comment by andrzejQ** [comment]
Nowa wersja komentarzy z użyciem `giscus -> GitHub Discussions`.
